spec/system/candidate_interface/candidate_apply_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'A candidate applying from Find' do
+  let(:provider_code) { '1AB' }
+  let(:course_code) { '2ABC' }
+  before do
+    visit "/candidate/apply" \
```

Comment:
Should we use the URL helpers here? `candidate_apply_path(providerCode: provider_code, ...)`

Reply:
I think it's fine to use the string in the test for the routes, it is more readable, and I think this practice is already spread in the project, but happy to change if someone has a strong opinion :)

Reply:
I think I prefer URL helpers actually in this case, though I agree that plain string is more obvious